plonky3/src/stark.rs

```diff
@@ -181,7 +191,9 @@ impl<T: FieldElement> Plonky3Prover<T> {
 
         let publics = circuit.get_public_values();
```

Comment:
here we only have phase 0 witness, so we can't get all the public value in general